{
    "type": "bundle",
    "id": "bundle--4f753b98-f2ee-4e47-adfd-4d0319fe7b2c",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--deca39ca-bdc2-470d-a3b7-b24adc43e639",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-31T01:20:13.097913Z",
            "modified": "2025-05-31T01:20:13.097913Z",
            "name": "CVE-2025-48887",
            "description": "vLLM, an inference and serving engine for large language models (LLMs), has a Regular Expression Denial of Service (ReDoS) vulnerability in the file `vllm/entrypoints/openai/tool_parsers/pythonic_tool_parser.py` of versions 0.6.4 up to but excluding 0.9.0. The root cause is the use of a highly complex and nested regular expression for tool call detection, which can be exploited by an attacker to cause severe performance degradation or make the service unavailable. The pattern contains multiple nested quantifiers, optional groups, and inner repetitions which make it vulnerable to catastrophic backtracking. Version 0.9.0 contains a patch for the issue.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-48887"
                }
            ]
        }
    ]
}